packages/scandipwa/src/component/FieldSelect/FieldSelect.style.scss

```diff
@@ -73,7 +73,7 @@ $select-arrow-width: 6px !default;
         border-style: solid;
         border-color: transparent;
 
-        overflow-y: scroll;
+        overflow-y: hidden;
```

Comment:
So now the options are not scrollable at all? What if I open the country select with 200+ options?

Reply:
Maybe try playing the scrollbar appearance instead?